src/objective-c/examples/SwiftSample/ViewController.swift

```diff
@@ -70,18 +70,18 @@ class ViewController: UIViewController {
 
     let requestsWriter = GRXWriter(value: request.data())
 
-    let call = GRPCCall(host: RemoteHost, path: method.HTTPPath, requestsWriter: requestsWriter)
+    let call = GRPCCall(host: RemoteHost, path: method?.httpPath, requestsWriter: requestsWriter)
```

Comment:
Instead of `method?.`, make `method` non-optional by adding `!` at the end of its initialization expression (`GRPCProtoMethod(...`)